samples/nextjs/jss-create.js

```diff
@@ -43,15 +49,69 @@ module.exports = function createJssProject(argv, nextSteps) {
   return nextSteps;
 };
 
+function getPath(filepath) {
+  return path.join(__dirname, filepath);
+}
+
+function cleanUpStrippedFiles() {
+  const files = getStrippedFiles();
+
+  console.log(chalk.cyan('Removing stripped files...'));
+
+  files.forEach(filepath => fs.unlinkSync(filepath))
+}
+
+function getStrippedFiles() {
+  return [
+    getPath('scripts/bootstrap.stripped.ts'),
+    getPath('scripts/scaffold-component.stripped.ts'),
+    getPath('src/lib/sitemap-factory.stripped.ts'),
+    getPath('next.config.base.stripped.js'),
+  ];
+}
+
+function createEmptyStarter() {
+  const dataDir = getPath('data');
+  const disconnectedProxyScript = getPath('scripts/disconnected-mode-proxy.ts');
+  const manifestTemplate = getPath('scripts/templates/component-manifest.ts');
+  const definitionsDir = getPath('sitecore/definitions');
+  const componentsDir = getPath('src/components');
+
+  console.log(chalk.cyan('Cleaning up the sample...'));
+
+  const strippedFiles = getStrippedFiles();
+
+  strippedFiles.forEach(strippedFilePath => {
+    const defaultFilePath = strippedFilePath.replace('.stripped', '');
+    fs.unlinkSync(defaultFilePath)
+    fs.renameSync(strippedFilePath, defaultFilePath);
+  })
+
+  fs.rmdirSync(dataDir, { recursive: true });
+  fs.unlinkSync(disconnectedProxyScript);
+  fs.unlinkSync(manifestTemplate);
+  fs.rmdirSync(definitionsDir, { recursive: true });
```

Comment:
Removing the `config.js` under `sitecore/definitions` breaks `jss manifest` and `jss deploy component` commands. Is this something we need to keep intact?

Reply:
I expected that we will not push items to Sitecore. In this case I don't create even `definition` file when you run `jss scaffold`